### PR TITLE
CAT-2476 Fix CloneBrick grouping functionality

### PR DESCRIPTION
**Pull request by sjaindl**
As this fix is not only relevant to feature/NOLB this fix should be merged into develop, too.
see #2374

### CODE REVIEW

**Comment by cheidenreich** [comment]
please run espresso tests

File: catroid/src/androidTest/java/org/catrobat/catroid/test/content/bricks/BrickCloneTest.java [hunk 1]
```diff
@@ -82,11 +85,40 @@
...
+	private Sprite secondSprite;
 
 	@Override
 	protected void setUp() throws Exception {
 		super.setUp();
 		sprite = new SingleSprite("testSprite");
+		secondSprite = new SingleSprite("secondSprite");
+	}
+
+	public void testSameSpriteReferenceAfterCloning() {
```
Comment (cheidenreich) [inline catroid/src/androidTest/java/org/catrobat/catroid/test/content/bricks/BrickCloneTest.java]: in my opinion this doesn't really test the clone brick

Comment (sjaindl) [inline catroid/src/androidTest/java/org/catrobat/catroid/test/content/bricks/BrickCloneTest.java]: > please run espresso tests
Is there already a Jenkins job for running the espresso tests that I can start?

Comment (sjaindl) [inline catroid/src/androidTest/java/org/catrobat/catroid/test/content/bricks/BrickCloneTest.java]: Well, I think it doesn't specifically test the clone brick .. but this problem isn't only relevant to the clone brick (also to the PointToBrick - that's also why I've made an interface for these bricks). It tests whether the method refreshSpriteReferences() really does what it should do. It should set the correct reference to a sprite in these bricks (e.g. after shallow cloning when there is still the old sprite reference). 
That was my intention for this test ;)

Comment (sjaindl) [inline catroid/src/androidTest/java/org/catrobat/catroid/test/content/bricks/BrickCloneTest.java]: But maybe I should move it to another test class..

**Comment by sjaindl** [comment]
ok to test

**Comment by cheidenreich** [comment]
please run espresso tests

**Pull request merged**